### PR TITLE
Adding initial template content

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Here's a first pass at the content for a working group template. It would probably be good to have some of the @opencontainers/tob members review it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+## Issues
+
+Issues are welcome for discussing proposed changes or asking questions.
+
+## Code style
+
+This project attempts to following principles:
```
Comment (samuelkarp) [inline CONTRIBUTING.md]:
```suggestion
This project attempts to follow these principles:
```

Having a hard time parsing the sentence, so I'm not sure if my suggestion has the same meaning.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- Linters and other style formatting tools are used, please run `make` before committing changes.
```
Comment (samuelkarp) [inline CONTRIBUTING.md]:
```suggestion
- Linters and other style formatting tools are used, please run `make lint` before committing changes.
```

While `default: lint` is specified in the `Makefile`, WG owners may decide to pick a different default target.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Pull requests
+
+PR's are welcome following the below guides:
```
Comment (samuelkarp) [inline CONTRIBUTING.md]:
```suggestion
PRs are welcome following the below guides:
```

(nit: no apostrophe for plurals)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+- For anything beyond a minor fix, opening an issue is suggested to discuss possible solutions.
+- Changes should be rebased on the main branch.
+- Changes should be squashed to a single commit.
```
Comment (samuelkarp) [inline CONTRIBUTING.md]:
```suggestion
- Changes should be squashed to a single commit per logical change.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+All changes must be signed (`git commit -s`) to indicate you agree to the [Developer Certificate or Origin](https://developercertificate.org/):
```
Comment (samuelkarp) [inline CONTRIBUTING.md:21]: We may want a workflow or app enforcing DCO here.

Comment (sudo-bmitch) [reply CONTRIBUTING.md:21]: I believe that is enabled by default for the organization. Or at least it's enabled on this repo already and it was created without any template.

**Comment by sudo-bmitch** [comment]
@samuelkarp thanks for the review! I believe I fixed everything you saw, and now it's obvious which file I didn't copy from OCI. 😄

File: GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Reference Types Working Group Governance
+
+## Discussion
+
+- Discussions take place during [weekly meetings](./README.md#meetings) and in GitHub issues.
+
+Working group members and maintainers shall follow the [OCI code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
```
Comment (samuelkarp) [inline GOVERNANCE.md]: s/maintainers/owners/g?  Or "leaders"?  The charter says "leaders", the WG-TEMPLATE says "owners".

Comment (sudo-bmitch) [reply GOVERNANCE.md]: Owners works for me, the charter has both.

**Review by samuelkarp — APPROVED** [review]

**Comment by sudo-bmitch** [comment]
@amye it looks like feedback on this has settled down and it should be safe to merge.

**Pull request merged**